sklearn/metrics/tests/test_classification.py

```diff
@@ -2882,6 +2883,25 @@ def test_classification_metric_division_by_zero_nan_validaton(scoring):
     cross_val_score(classifier, X, y, scoring=scoring, n_jobs=2, error_score="raise")
 
 
+def test_prfs_docstring_consistency():
```

Comment:
I think it makes sense to have this moved to `test_docstrings_parameters.py`. WDYT?

Reply:
Yes you're right. Done.